assets/js/components/Loadpoints/PrioButton/PrioButton.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+	<button
+		v-if="!editable"
+		ref="tooltip"
+		type="button"
+		class="btn btn-sm btn-outline-secondary position-relative border-0 pt-0"
+		data-testid="loadpoint-prio-button"
+		:class="{ 'opacity-25': !editable }"
+		data-bs-toggle="tooltip"
+		data-bs-html="true"
+		:title="tooltipTitle"
+		:aria-label="editable ? $t('main.loadpointPrio.prioTooltip') : null"
+	>
+		<component :is="icon" :class="`icon icon--${size}`"></component>
+	</button>
+	<button
+		v-if="editable"
+		ref="tooltip"
+		type="button"
+		class="btn btn-sm btn-outline-secondary position-relative border-0 pt-0"
+		data-testid="loadpoint-prio-button"
+		:class="{ 'opacity-25': !editable }"
+		data-bs-toggle="tooltip"
+		data-bs-html="true"
+		:title="tooltipTitle"
+		:aria-label="editable ? $t('main.loadpointPrio.prioTooltip') : null"
+	>
```

Comment:
Select Element in Button ist ungültiges Markup. Hier brauchen wir den Button eigentlich gar nicht. Müssen nur schauen, dass sich das Styling/Touchtarget Button-Style verhält.

Reply:
Schau dir mal die Implementierung von `CustomSelect` an. Der Slot direkt direkt in ein `label` gerendert und ist damit bereits eine Interaktionsfläche.